blockmanager.go

```diff
@@ -1063,7 +1095,7 @@ func (b *blockManager) handleInvMsg(imsg *invMsg) {
 			break
 		}
 	}
-	imsg.peer.requestQueue = requestQueue
+	b.peers[imsg.peer.ID()].requestQueue = requestQueue
```

Comment:
If you implement my previous suggestion this just becomes:

```Go
peerState.requestQueue = requestQueue
```

So you avoid another map lookup.